core/src/main/java/com/graphhopper/storage/TurnCostStorage.java

```diff
@@ -224,14 +238,14 @@ public int getToEdge() {
 
         @Override
         public boolean get(BooleanEncodedValue booleanEncodedValue) {
-            intsRef.ints[0] = turnCosts.getInt(turnCostPtr() + TC_FLAGS);
-            return booleanEncodedValue.getBool(false, -1, edgeIntAccess);
+            edgeAccess.setInt(0, 0, turnCosts.getInt(turnCostPtr() + TC_FLAGS));
+            return booleanEncodedValue.getBool(false, 0, edgeAccess);
         }
 
         @Override
         public double getCost(DecimalEncodedValue encodedValue) {
-            intsRef.ints[0] = turnCosts.getInt(turnCostPtr() + TC_FLAGS);
-            return encodedValue.getDecimal(false, -1, edgeIntAccess);
```

Comment:
I thought using -1 made it a bit more clear that we are passing an unused value? In the code above we still use -1